services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Same as the previous comment, it's recommended to handle exceptions where a record is not found in the database. You can use Django's `get_object_or_404` function to raise a 404 error if the `MovieSession` object is not found.